currencyfair.yaml

```diff
@@ -5155,6 +5435,18 @@ components:
         application/problem+json:
           schema:
             $ref: "#/components/schemas/ApiForbiddenError"
+    RequestConflictWithCurrentState:
+      description: Indicates a request conflict with the current state of the target resource.
```

Comment:
```suggestion
      description: Indicates a request conflicts with the current state of the target resource.
```

Reply:
Actually both of these read OK so non blocker

Reply:
Yeah, I took it from https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409
```
The HTTP 409 Conflict response status code indicates a request conflict with current state of the target resource.
```
Thanks for looking!